src/components/Layout/HomeContent.js

```diff
@@ -915,6 +915,7 @@ function Example1() {
           isFromPackageImport={false}
           noShadow={true}
           noMargin={true}>
+          {/* eslint-disable-next-line react/no-unknown-property */}
```

Comment:
What's the unknown property? Is this a bug in the lint rule?

Reply:
Technically the `meta` attribute on a `<div>` is unsupported so I think the linter is technically correct here. I've moved this to `.eslintrc` to make it clearer that we are ignoring the rule that says you can't use the `meta` attribute with `<div>`s.